src/core/RecyclerListView.tsx

```diff
@@ -778,4 +780,9 @@ RecyclerListView.propTypes = {
     // For e.x. toolbar within CoordinatorLayout are overlapping the recyclerlistview.
     // This method exposes the windowCorrection object of RecyclerListView, user can modify the values in realtime.
     applyWindowCorrection: PropTypes.func,
+
+    // This can be used to hook an itemLayoutListener to listen to which item at what index is layout.
+    // To get the layout params of the item, you can use the ref to call method getLayout(index), e.x. : `this._recyclerRef.getLayout(index)`
+    // but there is a catch here, since there might be a pending relayout due to which the queried layout might not be precise.
+    onItemLayout: PropTypes.func,
```

Comment:
Also, add this prop to readme. Also, mention that this can be used as a proxy to items rendered kind of callback

Reply:
Done